File: 프로그래머스/unrated/120895. 인덱스 바꾸기/인덱스 바꾸기.py
def solution(my_string, num1, num2):
    answer = ''
    my_string = list(my_string)
    a = my_string[num1]
    b = my_string[num2]
    my_string[num1] = b
    my_string[num2] = a
    for i in my_string:
        answer += i
    return answer

File: 프로그래머스/lv0/120895. 인덱스 바꾸기/인덱스 바꾸기.java
import java.util.*;
class Solution {
    public String solution(String my_string, int num1, int num2) {
        StringBuilder answer = new StringBuilder();
        char[] ch = my_string.toCharArray();
        char tmp = ch[num1];
        ch[num1] = ch[num2];
        ch[num2] = tmp;
        for (char i : ch)
            answer.append(i);
        return answer.toString();
    }
}